developer-docs-site/docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: "Update Fullnode With New Devnet Releases"
-slug: "update-fullnode-with-new-devnet-releases"
+title: "Update Fullnode With New Releases"
+slug: "update-fullnode-with-new-releases"
 sidebar_position: 11
 ---
 
 # Update Fullnode With New Releases
 
-When `devnet` is wiped and updated with newer versions, you will need to update your fullnode as well. If you do not, your fullnode will not continue to synchronize with the network. To update your fullnode, follow these steps:
+This document outlines the process for updating you fullnode. For fullnodes running in `devnet`, an additional data wipe step is required to allow the fullnode to continue syncrhonizing with the network.
 
 ## If you built the fullnode from aptos-core source code
 
-1. Shutdown your fullnode.
+1. Stop your local public fullnode by running the below command:
+  ```bash
+  cargo stop aptos-node
+  ```
 
 2. Delete the data folder (the directory path is what you specified in the configuration file, e.g., `fullnode.yaml`).
```

Comment:
Steps 2, 3, 4, 5 and 6 are only required if the user is running a fullnode in `devnet`. The other networks, e.g., `testnet` and `mainnet` don't require these steps. Can we make this distinction? e.g., put something like `(Only required for `devnet` upgrade)` before the steps? Or alternatively, split these steps into a substep for `devnet` specifically?


Step 4 is required for all networks, but it depends on the branch we're deploying. Here it says use the `devnet` branch. But, if the user is running a `testnet` or `mainnet` fullnode, they need to use those network branches, respectively 😄 Can we mention this, too?